### PR TITLE
WR 1675 explore the labour market styling

### PR DESCRIPTION
**Pull request by rjkuyvenhoven**

### CODE REVIEW

File: src/config/sync/field.field.paragraph.action_cards_publication.field_publications.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ field_name: field_publications
 entity_type: paragraph
 bundle: action_cards_publication
 label: Publications
-description: ''
+description: 'This field is for internal use only and should not be edited.'
```
Comment (infojunkie) [inline src/config/sync/field.field.paragraph.action_cards_publication.field_publications.yml:14]: Is this intentional?

**Review by infojunkie — APPROVED** [review]
Lgtm, thanks, other than one question about the publication field description.

File: src/config/sync/field.field.paragraph.action_cards_publication.field_publications.yml (hunk 1 above)
Comment (rjkuyvenhoven) [reply src/config/sync/field.field.paragraph.action_cards_publication.field_publications.yml:14]: I wasn't able to come up with a better description at the time.  Wanted something to discourage content editors from editing the field.  If you have a better description, I can update.

**Review by infojunkie — APPROVED** [review]
Lgtm, thanks

**Pull request merged**